src/translations/en-US.json

```diff
@@ -190,12 +190,12 @@
         "limit": 0
     },
     "geo.ErrorMessageLocationMissing": {
-        "value": "Sorry, we can't display this insight",
+        "value": "No location in your insight",
         "comment": "",
```

Comment:
Please describe what "location" stands for here so it is clear that it relates to geo chart (map) and location of pushpin pushed to the map....

Reply:
Hi @petrjanu, I have added the comment as your suggestion.
Thanks